lib/utils/fs-utils.js

```diff
@@ -11,17 +10,37 @@ var Q = require('q');
 var t = require('tmp');
 var path = require('path');
 var mkdirp = require('mkdirp');
-
+var fs= require ('fs');
+var index=0;
```

Comment:
rename index to something more meanifull (atomeclyTmpfileindex or something like that)